src/main/java/core/basesyntax/service/ReportService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public interface ReportService {
+    String generateReport(Map<String, Integer> fruits);
```

Comment:
Consider adding Javadoc comments to the 'generateReport' method to describe its purpose, input parameters, and return value.